src/content/intro/introText.coffee

```diff
@@ -189,7 +189,7 @@ Job.IntroTakeOver::next = Page.IntroTakeOver = class IntroTakeOver extends Page
 
     -- Liana, perhaps sensing that I'm liable to keep ranting and raving and piling punishments on the hussy as long as that... thing... remains in my sight, banishes the bra with a spell.
   || bg="Inn/FrontDeskSex"
-    --> I nod to the slave to begin her fucking. The Holiday Inn is mine. I can set up some <em>proper</em> facilities inside it now, and send slaves into the outer world.
```

Comment:
This one is intentional. It's not the dark lady fucking her, it's the dark lady nodding to the people who are going to be giving the slave her fucking (as if it were a noun).

:P